SWLOR.Game.Server/Feature/GuiDefinition/ShipManagementDefinition.cs

```diff
@@ -310,6 +310,81 @@ public GuiConstructedWindow BuildWindow()
                             .SetWidth(32f);
                     });
 
+                    col.AddRow(row =>
+                    {
+                        row.AddLabel()
+                            .SetText("Configuration Slot")
+                            .SetHeight(20f);
+                    });
+
+                    col.AddRow(row =>
+                    {
+                        row.AddButtonImage()
+                         .BindImageResref(model => model.Configuration1Resref)
```

Comment:
Let's fix the tab formatting on this stuff. One tab for each dot method, new line in between elements.